src/mpl/include/mpl_base.h

```diff
@@ -116,6 +116,8 @@
     do { switch(0) { case 0: case (expr_): default: break; } } while (0)
 #endif
 
+#define MPL_COMPILE_TIME_ASSERT(cond_) MPL_static_assert(cond_, "MPL_COMPILE_TIME_ASSERT failure")
```

Comment:
I think the addition of this new macro in MPL should be part of the previous MPL commit.

Reply:
The single `cond_` assert are added only to replace `{CH4,OFI}_COMPILE_TIME_ASSERT`, therefore I found it more relevant to group them together in a single commit. But I do not insist.

Reply:
Fair enough. I'll update it later.